__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const request = require("supertest")
+const app = require("../db/app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET/API', () => {
+    test('200 OK: message', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.msg).toBe("All OK")
+        })
+    })
+})
```

Comment:
You don't need a "All OK" message for /api - you will have to change this for the next task as that endpoint asks for something different on the path of /api.